sdk/go.mod

```diff
@@ -1,7 +1,3 @@
-module github.com/chronosphereio/pulumi-chronosphere/sdk/v1
+module github.com/chronosphereio/pulumi-chronosphere/sdk
 
 go 1.22
-
-require (
```

Comment:
this will return in the next PR when we start to actually build the go sdk again